internal/mage/engine.go

```diff
@@ -6,12 +6,17 @@ import (
 	"os"
 	"os/exec"
 	"runtime"
+	"time"
 
 	"dagger.io/dagger"
 	"github.com/dagger/dagger/internal/mage/util"
 	"github.com/magefile/mage/mg" // mg contains helpful utility functions, like Deps
 )
 
+const (
+	EngineImageRef = "ghcr.io/dagger/engine:v0.3.3"
+)
+
```

Comment:
Not sure if this is intentional. If it is, can we do the same for the Python SDK in the same context?

https://github.com/dagger/dagger/blob/f47ac949b0acff657cf37c819c8ee4719d6f34fb/sdk/python/src/dagger/connectors/base.py#L19-L22

Reply:
Ah okay, so *this* line in `internal/mage/engine.go` is where we have the code for publishing the image, so here we can't hardcode the SHA.

For the Go SDK we actually do hardcode the SHA same as we do in python (in `sdk/go/client.go`).